packages/tests/web-test-runner.config.js

```diff
@@ -16,33 +18,149 @@ const removeDevModeLogging = {
   },
 };
 
-const plugins = [];
-if (process.env.TEST_PROD_BUILD) {
-  console.log("Using production builds");
-  plugins.push(fromRollup(resolveRemap)(prodResolveRemapConfig));
-} else {
-  console.log("Using development builds");
-  plugins.push(fromRollup(resolveRemap)(devResolveRemapConfig));
-  plugins.push(removeDevModeLogging);
+function getPlugins() {
+  let resolveRemapConfig;
+  if (process.env.TEST_PROD_BUILD) {
+    console.log("Using production builds");
+    resolveRemapConfig = prodResolveRemapConfig;
+  } else {
+    console.log("Using development builds");
+    resolveRemapConfig = devResolveRemapConfig;
+  }
+  return [
+    fromRollup(resolveRemap)(resolveRemapConfig),
+    removeDevModeLoggingPlugin,
+    // Detect browsers without modules (e.g. IE11) and transform to SystemJS
+    // (https://modern-web.dev/docs/dev-server/plugins/legacy/).
+    legacyPlugin,
```

Comment:
Is this being exercised yet? Firefox ESR has modules. Maybe add it later?

Reply:
We know we'll need it, might as well be running it? Would rather notice problems it's introducing sooner, and it shouldn't add any latency. It should only take effect automatically on browsers that don't support modules.